charms/istio-gateway/tests/unit/test_charm.py

```diff
@@ -236,3 +236,21 @@ def test_manifests_applied_with_replicas_config(configured_harness, mocked_clien
     # Assert
     assert gateway_deployment["spec"].get("replicas") == replicas_config_value
     assert configured_harness.charm.model.unit.status == ActiveStatus("")
+
+
+def test__worklod_service_annotations(configured_harness, mocked_client):
```

Comment:
typo

Reply:
Hahaha, it looks like a pretty consistent typo